nautobot_chatops/dispatchers/webex.py

```diff
@@ -13,26 +13,42 @@
 logger = logging.getLogger("rq.worker")
 
 # Create a metric to track time spent and requests made.
-BACKEND_ACTION_LOOKUP = backend_action_sum.labels("webex_teams", "platform_lookup")
-BACKEND_ACTION_MARKDOWN = backend_action_sum.labels("webex_teams", "send_markdown")
-BACKEND_ACTION_BLOCKS = backend_action_sum.labels("webex_teams", "send_blocks")
-BACKEND_ACTION_SNIPPET = backend_action_sum.labels("webex_teams", "send_snippet")
+BACKEND_ACTION_LOOKUP = backend_action_sum.labels("webex", "platform_lookup")
+BACKEND_ACTION_MARKDOWN = backend_action_sum.labels("webex", "send_markdown")
+BACKEND_ACTION_BLOCKS = backend_action_sum.labels("webex", "send_blocks")
+BACKEND_ACTION_SNIPPET = backend_action_sum.labels("webex", "send_snippet")
 
 # pylint: disable=abstract-method
 
 
-class WebExTeamsDispatcher(AdaptiveCardsDispatcher):
-    """Dispatch cards and messages to WebEx Teams."""
+class WebExDispatcher(AdaptiveCardsDispatcher):
+    """Dispatch cards and messages to WebEx."""
 
-    platform_name = "WebEx Teams"
-    platform_slug = "webex_teams"
+    platform_name = "WebEx"
+    platform_slug = "webex"
 
     platform_color = "6EBE4A"
 
     def __init__(self, *args, **kwargs):
-        """Init a WebExTeamsDispatcher."""
+        """Init a WebExDispatcher."""
         super().__init__(*args, **kwargs)
-        self.client = WebexTeamsAPI(access_token=settings.PLUGINS_CONFIG["nautobot_chatops"]["webex_teams_token"])
+        # v1.4.0 Deprecation warning
```

Comment:
This is duplicated code, you might want to make a function for it.

Reply:
This is in a couple of places across the webex files when initially loading the token when initializing the class, but I think since this is just for a deprecation warning where it will be removed at a later date, it should be ok to leave as is.